README.md

```diff
@@ -1 +1,65 @@
-# siddhi-io-mqtt
\ No newline at end of file
+siddhi-io-mqtt
+======================================
+
+The **siddhi-io-mqtt** is an extension to <a target="_blank" href="https://wso2.github.io/siddhi">Siddhi</a> mqtt source and sink implementation,that publish and receive events from mqtt broker.
+
+Find some useful links below:
+
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt">Source code</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt/releases">Releases</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt/issues">Issue tracker</a>
+## Latest API Docs 
+
+## How to use 
+
+**Using the extension in <a target="_blank" href="https://github.com/wso2/product-sp">WSO2 Stream Processor</a>**
+
+* You can use this extension in the latest <a target="_blank" href="https://github.com/wso2/product-sp/releases">WSO2 Stream Processor</a> that is a part of <a target="_blank" href="http://wso2.com/analytics?utm_source=gitanalytics&utm_campaign=gitanalytics_Jul17">WSO2 Analytics</a> offering, with editor, debugger and simulation support. 
+
+* This extension is shipped by default with WSO2 Stream Processor, if you wish to use an alternative version of this extension you can replace the component <a target="_blank" href="https://github.com/wso2-extensions/siddhi-execution-unique/releases">jar</a> that can be found in the `<STREAM_PROCESSOR_HOME>/lib` directory.
+
+**Using the extension as a <a target="_blank" href="https://wso2.github.io/siddhi/documentation/running-as-a-java-library">java library</a>**
+
+* This extension can be added as a maven dependency along with other Siddhi dependencies to your project.
+
+```
+     <dependency>
+          <groupId>org.wso2.extension.siddhi.io.mqtt</groupId>
+          <artifactId>siddhi-io-mqtt-parent</artifactId>
+           <version>x.x.x</version>
+     </dependency>
+```
+
+## Jenkins Build Status
+
+---
+
+|  Branch | Build Status |
+| :------ |:------------ |
+| master  | [![Build Status](https://wso2.org/jenkins/job/siddhi/job/siddhi-io-mqtt/badge/icon)](https://wso2.org/jenkins/job/siddhi/job/siddhi-io-mqtt/) |
+
+---
+
+## Features
+
+
+## How to Contribute
+ 
+  * Please report issues at <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt/issues">GitHub Issue Tracker</a>.
+  * Send your bug fixes pull requests to <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt/tree/master">master branch</a>.
+ 
+## Contact us 
+
+ * Post your questions on <a target="_blank" href="http://stackoverflow.com/">Stackoverflow</a> with the tag <a target="_blank" href="http://stackoverflow.com/search?q=siddhi">"siddhi"</a>. 
```

Comment:
Make sentence as uniform as unique extesnion

Reply:
fixed in 3ba15c9